src/core/lib/iomgr/work_serializer.cc

```diff
@@ -43,10 +59,11 @@ class WorkSerializer::WorkSerializerImpl : public Orphanable {
   void Orphan() override;
 
  private:
+  void DrainQueueOwned();
+
   // An initial size of 1 keeps track of whether the work serializer has been
   // orphaned.
-  std::atomic<size_t> size_{1};
-  std::atomic<bool> draining_{false};
+  std::atomic<size_t> refs_{MakeRefPair(0, 1)};
```

Comment:
This should probably be `std::atomic<uint64_t>`.

Reply:
You're right. Thanks!